src/__tests__/operate.test.js

```diff
@@ -0,0 +1,16 @@
+import operate from '../logic/operate';
+
+describe('operate', () => {
+  test('shoud return 10 after adding 7 + 3', () => {
+    expect(operate('7', '3', '+')).toEqual('10');
+  });
+  test('shoud return 5 after adding 8 - 3', () => {
+    expect(operate('8', '3', '-')).toEqual('5');
+  });
+  test('shoud return 30 after adding 6 x 5', () => {
+    expect(operate('6', '5', 'x')).toEqual('30');
+  });
+  test('shoud return 4 after adding 12 ÷ 3', () => {
+    expect(operate('12', '3', '÷')).toEqual('4');
+  });
+});
```

Comment:
- [ ] As required for the project to cover all the functionalities & scenarios of the `operate.js` and `calculate.js` modules with tests. I think you miss some of them such as the `modulus` operation, `can't divide by 0 error`,...
You can check again your `operate.js` file and make sure to add the remaining tests scenarios